packages/lib/environment/auth.ts

```diff
@@ -0,0 +1,31 @@
+import { prisma } from "@formbricks/database";
+import { Session } from "next-auth";
+
+export const hasUserEnvironmentAccess = async (user: Session["user"], environmentId: string) => {
```

Comment:
please use `userId` instead of the user object to make it better reusable for different occasions.

Reply:
done